### PR TITLE
Fix uncaught promise exception for bindAudioOutput API

### PR DESCRIPTION
**Pull request by anuranduttaroy**
**Issue #:** https://github.com/aws/amazon-chime-sdk-js/issues/907

**Description of changes:** Checking for `browserBehavior.supportsSetSinkId()` before calling `AsyncScheduler() -> this.bindAudioOutput();` in [`DefaultDeviceController`](https://github.com/aws/amazon-chime-sdk-js/blob/59bf60c30c8450451d9d1e36ec30a5b67e787607/src/devicecontroller/DefaultDeviceController.ts#L402)

**Testing**

1. Have you successfully run `npm run build:release` locally?
Yes.

2. How did you test these changes?
Tested by running the `serverless demo` & the `meeting readiness checker` demo in Firefox & Safari browser to check if there is any issue while
    - Starting/Joining a meeting.
    - Choosing audio input device, mute, unmute operation.
    - Enable/Disable speaker from the meeting canvas.
    - Set the `media.setsinkid.enabled` preference to `true` in Firefox. This enables the use of `SetSinkId` API as mentioned here https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/setSinkId and verified this error does not show up in the browser console.

3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it?
Yes. Serverless demo.

4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
NA

5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
NA

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: guides/07_FAQs.md [hunk 1]
```diff
@@ -59,6 +59,12 @@ Note that due to limitations in transpilers, requirements of the web platform mi
 
 [Android Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=635686&sort=-stars&q=android%20chrome%20bluetooth%20headphone&can=2) has a known issue switching between bluetooth audio output devices. While clients can continue the meeting using the default device, there is a [bug](https://bugs.chromium.org/p/chromium/issues/detail?id=635686&sort=-stars&q=android%20chrome%20bluetooth%20headphone&can=2) related to switching to an bluetooth audio output.
 
+### I am getting `Cannot select audio output device. This browser does not support setSinkId` error on the browser console. Is this a known issue?
+
+In the background, `bindAudioElement()`, `bindAudioStream()` and `bindAudioDevice()` call the browser API [`setSinkId()`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/setSinkId). The full list of browsers that support `setSinkId` API is given here https://caniuse.com/?search=setsinkid. In Firefox, this feature is behind the `media.setsinkid.enabled` preference (needs to be set to `true`). To change preferences in Firefox, visit `about:config`
```
Comment (richnew10) [inline guides/07_FAQs.md:64]: Nit: finish sentence with `.`.

Nit: use the Oxford comma in the list in the first sentence.

Nit: colon before hyperlink, but prefer not having a bare hyperlink at all.

File: guides/07_FAQs.md [hunk 2]
```diff
@@ -59,6 +59,12 @@ Note that due to limitations in transpilers, requirements of the web platform mi
...
+
+If you wish to find whether the browser supports `setSinkId()`, you can use the `supportsSetSinkId()` helper function in BrowserBehavior.
```
Comment (richnew10) [inline guides/07_FAQs.md:66]: "Use `BrowserBehavior.supportsSetSinkId()` to determine whether the browser supports `setSinkId()` before calling these methods."

File: guides/07_FAQs.md [hunk 3]
```diff
@@ -59,6 +59,12 @@ Note that due to limitations in transpilers, requirements of the web platform mi
 
 [Android Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=635686&sort=-stars&q=android%20chrome%20bluetooth%20headphone&can=2) has a known issue switching between bluetooth audio output devices. While clients can continue the meeting using the default device, there is a [bug](https://bugs.chromium.org/p/chromium/issues/detail?id=635686&sort=-stars&q=android%20chrome%20bluetooth%20headphone&can=2) related to switching to an bluetooth audio output.
```
Comment (richnew10) [inline guides/07_FAQs.md:60]: While you're here: s/an bluetooth/a Bluetooth

**Review by richnew10 — CHANGES_REQUESTED** [review]

File: guides/07_FAQs.md (hunk 1 above)
Comment (anuranduttaroy) [reply guides/07_FAQs.md]: Fixed

File: guides/07_FAQs.md (hunk 2 above)
Comment (anuranduttaroy) [reply guides/07_FAQs.md]: Fixed

File: guides/07_FAQs.md (hunk 3 above)
Comment (anuranduttaroy) [reply guides/07_FAQs.md]: Fixed

**Review by richnew10 — APPROVED** [review]

**Comment by gustavofamorim** [comment]
Hi all.
You guys know or have an idea about when this fix should be released?

**Comment by rusicsemenov** [comment]
Hello, after update to `" amazon-chime-sdk-js ":" ^ 2.2.0 ",` i steal have issue with `This browser does not support setSinkId`. Did you fix it?

**Comment by rnewman** [comment]
@rusicsemenov what API are you calling? What is the full error?

If you’re calling `bindAudioDevice`, you need to call `supportsSetSinkId` first to decide whether the browser supports it.

**Comment by rusicsemenov** [comment]
But I get the error message: 
`[Log] Error: Cannot select audio output device. This browser does not support setSinkId. (app.js, line 25)` 
when I bind ellement 
```js
await meetingSession.audioVideo.bindAudioElement(
    document.getElementById('meeting-audio')
);
```

**Pull request merged**